src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java

```diff
@@ -13,15 +10,12 @@
 
     private ArrebolJobState jobState;
 
-    private Map<String, ArrebolTask> tasks;
+    private List<ArrebolTask> tasks;
 
     public ArrebolJob(String label, Collection<ArrebolTask> tasks){
         this.jobState = ArrebolJobState.SUBMITTED;
-        this.tasks = new HashMap<>();
-
-        for(ArrebolTask task: tasks) {
-            this.tasks.put(task.getId(), task);
-        }
+        this.tasks = new LinkedList<>();
+        this.tasks.addAll(tasks);
```

Comment:
has LList a constructor that receives another collection as arg?

Reply:
Done.